src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -150,6 +150,14 @@ public static void merge(List<RepoConfiguration> repoConfigs, List<AuthorConfigu
 
             branchMatchingRepoConfig.addAuthors(authorConfig.getAuthorList());
         }
+
```

Comment:
Why is this part not included in the for loop above?

Reply:
Because the repoConfigs in the list use different AuthorConfig objects than the ones passed into this function, and the first loop only loop through the AuthorConfig list